presto-parser/src/main/java/com/facebook/presto/sql/tree/ArithmeticUnaryExpression.java

```diff
@@ -38,12 +51,12 @@ public ArithmeticUnaryExpression(Sign sign, Expression value)
 
     public static ArithmeticUnaryExpression positive(Expression value)
```

Comment:
I believe this need to have SqlLocation. It's a factory method called directly from `AstBuilder`